update signing information (#56)
ghga-de/docs.ghga.de: user_docs/user_stories/submission/dpc_preparation.md

- [x] Consider the additional information that is required. 
      - A number of the annexes to the GHGA Data Processing Contract require information from the Data Controller. It is important for the Data Controller to consider in advance how they will respond.

- [x] Submit a [Pre-Submission Inquiry](https://www.ghga.de/about-us/presubmission-enquiries) or contact us via the <general:GHGA Helpdesk>. GHGA will send you feedback whether or not the intended submission can be supported.

- [x] Determine the authorised signatory person for the Data Controller.
      - Different institutions will define differently which roles are authorised to sign contracts on their behalf. It is not typical that an individual researcher can alone sign a contract. 

- [x] Submit detailled information to complete the DPC.

>       - If the proposed data submission can be supported by GHGA, the Data Controller will be asked by GHGA to supply the information necessary to prepare a bespoke Data Processing Contract for the Data Controller. THis includes details like the adress of the institutions, signatories, persons authorized to act and a description of the data to be submitted.

Reviewer: ```suggestion
      - If the proposed data submission can be supported by GHGA, the Data Controller will be asked by GHGA to supply the information necessary to prepare a bespoke Data Processing Contract for the Data Controller. This includes details like the address of the institutions, signatories, persons authorised to act, and a description of the data to be submitted.
```

- [x] Submit the signed Data Processing Contract to GHGA for counter-signing and propossing. 

### Details on Signing Procedure

- The signing can be carried out either using "wet ink" or a qualified electronic signature:

>       - For wet ink signing (preferred), submit two physical copies of the signed contract should be sent to GHGA Central to the following adress:

Reviewer: ```suggestion
      - For wet ink signing (preferred), submit two physical copies of the signed contract to GHGA Central using the following address:
```
